src/common/models/Cookie.class.js

```diff
@@ -245,7 +247,9 @@ class Cookie {
     this.session.gar.voyageDeparture.departurePort = voyageObj.departurePort;
     this.session.gar.voyageDeparture.departureLat = trimToDecimalPlaces(voyageObj.departureLat, 6);
     this.session.gar.voyageDeparture.departureLong = trimToDecimalPlaces(voyageObj.departureLong, 6);
-    this.session.gar.voyageDeparture.departurePortChoice = voyageObj.portChoice || (voyageObj.departureLat || voyageObj.departureLong) ? 'No' : 'Yes';
+
```

Comment:
you could move some of the logic from the getters and setters into a helper method if you want to, think this is outside the scope of the ticket though

Reply:
Has passed e2e tests (with a different version to handle other bugs).
* A mannual test found it is ok.